requirements_distributed.txt

```diff
@@ -8,6 +8,10 @@ tensorboardX<2.3
 GPUtil
 tblib
 awscli
+modin[ray]
```

Comment:
We had modin in extras since it's not a default option. Do we need it here? Can we keep it in `requirements_extra.txt`?